src/redux/wallets/actions.ts

```diff
@@ -155,7 +156,9 @@ export const getBalance = (
 export const sendTransferTransaction = (
     account: IAccountState,
     toAddress: string,
-    amount: string
+    amount: string,
+    gasPrice: BigNumber,
```

Comment:
we should do this more generic, use an object here, `feeOptions : {gasPrice, gasLimit}`